### PR TITLE
Nepal GloFAS stations

### PR DESCRIPTION
**Pull request by turnerm**
An analysis to select which stations in Nepal to use

### CODE REVIEW

File: analyses/npl/01_get_stations.md [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+    # Select only neeeded columns
+    table = table[[1, 2, 3, 4, 5, 7] + cnums_lat_lon[i]]
+    table.columns = cnames
+    table = gpd.GeoDataFrame(
+        table, 
+        geometry=gpd.points_from_xy(
+            table['lon'], table['lat']))
+    table_list.append(table)
+    
+df_glofas = pd.concat(table_list, ignore_index=True).set_crs("EPSG:4326")
```
Comment (hannahker) [inline analyses/npl/01_get_stations.md:90]: Generally, probably best to pull the crs from a global variable

File: analyses/npl/01_get_stations.md [hunk 2]
```diff
@@ -0,0 +1,379 @@
+```python
+import os
+from pathlib import Path
+import requests
+import yaml
+
+import tabula
```
Comment (hannahker) [inline analyses/npl/01_get_stations.md:7]: Should add tabula and selenium to the ```requirements.txt```

File: analyses/npl/01_get_stations.md [hunk 3]
```diff
@@ -0,0 +1,379 @@
...
+    'Betrawati': 'Trishuli at Betrawati',
+    'Rajaiya': 'Rajaiya (rainfall station)',
+    'Lothar': 'Lothar Khola at Lothar',
+    'Jalbire': 'Balefi at Jalbire',
+    'Rasnalu': 'KhimtiKhola at Rasnalu',
+    'Chatara': 'Saptakoshi at Chatara (old)'
+}
+```
+
+I give up, will find another way to get the basin
```
Comment (hannahker) [inline analyses/npl/01_get_stations.md:379]: Hahaha...

Comment (hannahker) [reply analyses/npl/01_get_stations.md:379]: Not sure if it's helpful at this point but we used this for fuzzy matching of admin area names in Bangladesh: https://github.com/seatgeek/fuzzywuzzy

**Review by hannahker — APPROVED** [review]
Looks good to me :) Just a couple of minor comments.

File: analyses/npl/01_get_stations.md (hunk 1 above)
Comment (turnerm) [reply analyses/npl/01_get_stations.md:90]: very good point, done!

File: analyses/npl/01_get_stations.md (hunk 2 above)
Comment (turnerm) [reply analyses/npl/01_get_stations.md:7]: Done!

File: analyses/npl/01_get_stations.md (hunk 3 above)
Comment (turnerm) [reply analyses/npl/01_get_stations.md:379]: Ah great, for next time ;)

**Pull request merged**